plugins/eni/commands/commands.go

```diff
@@ -78,6 +81,17 @@ func add(args *skel.CmdArgs, engine engine.Engine) error {
 	}
 	log.Debugf("Found ipv4 gateway and netmask for ENI: %s %s", ipv4Gateway, ipv4Netmask)
 
+	_, ipv4Net, err := net.ParseCIDR(fmt.Sprintf("%s/%s", ipv4Gateway, ipv4Netmask))
```

Comment:
Nit: store `fmt.Sprintf("%s/%s", ipv4Gateway, ipv4Netmask)` as a string and make both ipv4 and ipv6 checks semantically consistent